main.go

```diff
@@ -62,9 +63,14 @@ func main() {
 
 	log.Println("Nancy version: " + buildversion.BuildVersion)
 
-	if config.UseStdIn == true {
+	if config.UseStdIn && config.IQ {
+		doStdInAndParseForIQ(config)
+	}
+
+	if config.UseStdIn && !config.IQ {
 		doStdInAndParse()
-	} else {
+	}
+	if !config.UseStdIn && !config.IQ {
 		doCheckExistenceAndParse()
```

Comment:
Same goes for these in that it feels weird. See comment for next block....feels like we could maybe push the config.IQ check down into just where we call audit and not need it here.

Reply:
That is totally a great point!

Reply:
I think I kinda dealt with this a bit better in: https://github.com/sonatype-nexus-community/nancy/pull/68/commits/706eedc0682b1c5132999e07ee536aa9236ebc4f